commands/testdata/new_function/invalid3.yml

```diff
@@ -0,0 +1,10 @@
+proovider:
```

Comment:
Can you use an obviously deliberate invalid keys in testcases like this - `invalidprovider:` for example, makes it much clearer for readers.